openassessment/xblock/grade_mixin.py

```diff
@@ -584,3 +585,80 @@ def _assessment_grade_context(self, assessment):
                     part['option']['label'] = option_labels.get(option_label_key, part['option']['name'])
 
         return assessment
+
+    def _get_score_explanation(self, assessment_steps, score):
+        """
+        Return a string which explains how grade is calculated for the ORA assessment
+        based on assessment_steps.
+        Args:
+            assessment_steps (list): List of assessment_steps for some ORA.
+            score (dict): Dict containing points_possible, points_earned and its related attributes
+        Returns:
+            str
+        """
+
+        first_sentence = _(
+            "You have successfully completed this problem and received a {earned_points}/{total_points}."
```

Comment:
The sentence is not always this one. If the user didn't finish all the steps, we should display this status box too.
![2020-06-08_16-15](https://user-images.githubusercontent.com/27893385/84070919-445e1000-a9a3-11ea-98f0-b945f45b6508.png)

Reply:
The message box needs to be added alongside this: 
https://github.com/edx/edx-ora2/blob/685846f34743685636d9d03b94919c54ff2b62da/openassessment/templates/openassessmentblock/grade/oa_grade_incomplete.html#L27

Reply:
@giovannicimolin I think the document is confusing `incomplete` as `waiting`. 

`incomplete` mean that student haven't complete some assessment that he/she must to get to the next step e.g `peer assessment` or `self assessment`.

`waiting` mean that student have complete all assessment on his/her part but some assessment is left either on **peer** side or **staff** side.

I think that the message box should be added to the `waiting` html template instead of `incomplete` html template.

However, that is my understanding. I will add the message box to `incomplete` and if you think I am right, i will put it to `waiting` instead.

Thanks